core/state_processor.go

```diff
@@ -29,9 +31,14 @@ import (
 )
 
 var (
-	BlockReward = big.NewInt(5e+18)
-	big8        = big.NewInt(8)
-	big32       = big.NewInt(32)
+	MaximumBlockReward       = big.NewInt(5e+18) // that's shiny 5 ether
+	big8                     = big.NewInt(8)
+	big32                    = big.NewInt(32)
+	DisinflationRateQuotient = big.NewInt(4)
+	DisinflationRateDivisor  = big.NewInt(5)
+	DefaultEraLength         = big.NewInt(5000000) // Convenient for testing.
```

Comment:
Since we are using DefaultEraLength only in testing, perhaps we shouldn't export it,  and maybe only include it in the tests?